dockers/base-conda/Dockerfile

```diff
@@ -59,8 +59,8 @@ ENV \
     LD_LIBRARY_PATH="/root/miniconda3/lib:$LD_LIBRARY_PATH" \
     CUDA_TOOLKIT_ROOT_DIR="/usr/local/cuda" \
     MKL_THREADING_LAYER=GNU \
-    MAKEFLAGS="-j$(nproc)" \
-    # MAKEFLAGS="-j1" \
+    # MAKEFLAGS="-j$(nproc)" \
```

Comment:
```suggestion
```

Reply:
would be thinking to make it as argument as seems for GHA 2 is max to not crash the process but locally I want to use all